test/index.ts

```diff
@@ -129,6 +131,7 @@ describe('fs', () => {
 
   it('writeFile() - path is required', async () => {
     try {
+      // @ts-ignore
```

Comment:
Use `ts-expect-error` here. We are expecting the type to mismatch, instead of letting the compiler ignore the line.